sbi/inference/snpe/snpe_base.py

```diff
@@ -435,6 +425,22 @@ def _train(
         self._summary["epochs"].append(epoch)
         self._summary["best_validation_log_probs"].append(self._best_val_log_prob)
 
+    def _calibration_kernel_loss(
```

Comment:
This function does not compute the loss. It computes the log_prob. I suggest to add a minus to the returned value here.

Also, this is a super general function, deciding on what loss to use (MLE or corrected loss). The calibration kernel is just a small part of this function, and I would not put it in the function name. I suggest to call this something like `compute_loss()`.

Reply:
Or just `_loss`. Most things on a computer are... er... computed. ;)

Reply:
Also possible: `_calibrated_loss`.

Reply:
+1 for` _loss()`